### PR TITLE
Add documentation to 'python-remove'

### PR DESCRIPTION
**Pull request by ivan-podorozhnyi**
Added 'python-remove' function description and usage line
Affects #30

### CODE REVIEW

File: bin/pem-python-remove [hunk 1]
```diff
@@ -2,7 +2,14 @@
 [ "$PEM_DEBUG" = "yes" ] && set -x
 
 pem-default() {
+    [ "$1" = "-h" ] && usage && exit 0
     [ -z "$1" ] && echo "Please set a version to be deleted. " && exit -3
     echo "Remove '$1' version of Python from pyenv."
     pyenv uninstall -f "$1"
 }
+
+usage(){
```
Comment (extsoft) [inline bin/pem-python-remove:10]: @ivan-podorozhnyi Please order messages as described in task.

File: bin/pem-python-remove [hunk 2]
```diff
@@ -2,7 +2,14 @@
...
+    echo "Function removes Python's version."
+    echo "Usage: pem python-remove <version>"
```
Comment (extsoft) [inline bin/pem-python-remove]: @ivan-podorozhnyi add empty line after this message.

File: bin/pem-python-remove [hunk 3]
```diff
@@ -2,7 +2,14 @@
 [ "$PEM_DEBUG" = "yes" ] && set -x
 
 pem-default() {
+    [ "$1" = "-h" ] && usage && exit 0
```
Comment (extsoft) [inline bin/pem-python-remove]: @ivan-podorozhnyi don't do this. Please take a look for https://github.com/extsoft/pem/commit/90f87066d1a54814ea5134bd37429baa96555b83

**Review by extsoft — CHANGES_REQUESTED** [review]
@ivan-podorozhnyi thank you for contributing. Please fix comments.

**Comment by ivan-podorozhnyi** [comment]
@extsoft Ok, I will make changes accordingly

**Review by extsoft — APPROVED** [review]
@ivan-podorozhnyi thanks!

**Pull request merged**